bin/dbatools-index.json

```diff
@@ -4773,6 +4773,15 @@
                            "5000",
                            ""
                        ],
+                       [

```

Comment:
Welcome and thanks for the contribution!! Can you remove this file from the PR please - this one gets auto generated by the module.

Reply:
Yes, indeed thank you. Also, cool avatar!

Reply:
Thank you guys and sorry for the delay. Working on it now